tests/behat/bootstrap/FeatureContext.php

```
@@ -404,4 +404,51 @@ public function assertNoHorizontalOverflowAtWidth(string $width): void {
+   * scrollbar gutter while still catching real overflow that would force a
+   * horizontal scrollbar on a phone.
+   */
+  #[Then('the page reflows without horizontal scrolling at :width pixels wide')]
+  public function assertReflowsWithoutHorizontalScrolling(int $width): void {
+    $session = $this->getSession();
+    $session->resizeWindow($width, 900, 'current');
+
+    $overflow = (int) $session->evaluateScript('document.documentElement.scrollWidth - document.documentElement.clientWidth');
+
+    if ($overflow > 16) {
+      throw new \Exception(sprintf('The page scrolls horizontally by %dpx at %dpx wide.', $overflow, $width));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '450', column '17'). (undefined)

(MissingImport)